packages/caravan-bitcoin/package.json

```diff
@@ -77,7 +77,7 @@
     "bip32": "^2.0.5",
     "bip66": "^1.1.5",
     "bitcoin-address-validation": "^2.2.3",
-    "bitcoinjs-lib": "^5.1.10",
+    "bitcoinjs-lib-v5": "npm:bitcoinjs-lib@^5.1.10",
```

Comment:
caravan/bitcoin was getting confused and accidentally using the bitcoinjs-lib v6! same problem I was originally having in caravan/psbt